Sources/Helpers/SharingManager.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  SharingManager.swift
+//  OpenFoodFacts
+//
+//  Created by Егор on 2/23/18.
+//  Copyright © 2018 Andrés Pizá Bückmann. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class SharingManager {
```

Comment:
Will be great to use the keyword final if this class should not be inherit in future.